### ISSUE TITLE
Menus with ScrollView can not scroll

### ISSUE THREAD

**Issue by xiaoxidong** [aheze/Popovers#42]
Put a ScrollView inside Menus and can not scroll.

**Comment by aheze** [comment]
Yeah, currently menus don't work with ScrollViews. I am working on v1.4.0 though — should be out in a couple days. In that release I can probably get scroll view support working.

**Comment by richardpineo** [comment]
It seems like development on this project has fallen off? 

I'm faced with this issue where my popover contents requires a ScrollView - any ideas on how to get this to work?

(What I'm seeing is that the scrollview does appear and function but the taps often 'go through' the contents to the underlying scrollview)

**Comment by aheze** [comment]
Yeah, I've been working on some other projects... maybe just use a `.sheet` instead. Popovers is meant for more lightweight views like tips and notifications

**Comment by richardpineo** [comment]
Makes sense. Still, do you have any insight on this functionality? Maybe there is an easy fix or direction you could point me.

I'm using popovers to present a menu-like (simple VStack) filtering that applies to a map. Without a ScrollView for the menu contents, everything works fine. When a ScrollView is added, half of the time the drag gesture scrolls the scroll view and tap gestures allow menu items can be selected as expected but the other half of the time the tap or drag gesture 'passes through' to the underlying view. 

I am not doing anything fancy here and cannot figure out the pattern for why sometimes taps work and others they do not.

No worries if you don't have bandwidth for this but thought I'd ask.

![image](https://github.com/aheze/Popovers/assets/8593979/720e517a-6e89-40b5-9b3f-2b1195a1960c)
